Intro to coding - Improved (#2716)
enkidevs/curriculum: coding-intro/coding-intro-core/coding-intro-introduction/intro-to-coding.md

---
## Content

Whether you want to make an omelet or a crème brûlée, you must know how to crack an egg. Similarly, whether you want to edit a spreadsheet or send a man to the moon, this is where you begin, where it all begins. We'll start with the building blocks of any and all programming languages. When combined, they can create any program:

> - **variables** - to store information like x=5

Reviewer: ```suggestion
- **variables** - to store information like `x = 5`
```

- **data types** - to differentiate between numbers, text, etc.
- **functions** - to make little blocks for different tasks
- **conditions** - to make yes/no decisions
- **loops** - to repeat tasks as we please

> Over the following lessons, we'll cover these fundamentals, a few key themes, some helpful tools, and you will be ready to cook!

Reviewer 2: Thoughts on being a bit more specific? Maybe mentioning the "robot" is a bit more fun :shrug: 

```suggestion
Over the following lessons, we'll cover these fundamentals, a few key themes, some helpful tools, and you'll even teach a robot to cook!
```

Author: I was just finishing the metaphor that we started on the first line. I think the robot might be coming out of nowhere for them.

Author: I'd personally keep it like this. Let me know if you feel like that is lacking something.
